ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -905,12 +911,20 @@ TableHeaderUiFacadeImpl getTableHeaderUiFacade() {
   protected TableColumnExt newTableColumnExt(int modelIndex) {
     TableColumnExt result = new TableColumnExt(modelIndex);
     Class<?> columnClass = getTreeTableModel().getColumnClass(modelIndex);
-    TableCellRenderer renderer = createCellRenderer(columnClass);
+    String columnName = getTreeTableModel().getColumnName(modelIndex).toLowerCase();
+    TableCellRenderer renderer;
+    Boolean costColumn = false;
+    if (Double.class.equals(columnClass) && columnName.equals("cost")) {
```

Comment:
This will work only if your system locale is English. Should it be some other locale, column name will not be "cost".

You can find TaskDefaultColumn instance by model index: `TaskDefaultColumn.values()[modelIndex]`